app/services/weather/check_weather_for_city_service.rb

```diff
@@ -2,20 +2,32 @@
 
 module Weather
   class CheckWeatherForCityService
+    STORE_TIME = 1.day.freeze
```

Comment:
cities are not going to be moving so this could be extended even further if needed